Form-Controls/index.html

```diff
@@ -13,15 +13,57 @@
     <h1>Product Pick</h1>
   </header>
   <main>
-  <form>
+  <form class="form" action="reesults.html" method="GET">
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
-  </form>
+     <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" required>
+     </div>
+     <div>
+        <label for="email">Email</label>
+        <input type="email" id="email" required>
+     </div>
+     
+     <div>
+      <label for="date">Delivery Date</label>
+      <input type="date" id="date" min="2024-11-10">
```

Comment:
Perfect! You set min for delivery date. How we can be sure that it could be until next 4 weeks? (Based on the criteria of the Form in [Readme](https://github.com/RihannaP/Module-User-Focused-Data/blob/main/Form-Controls/README.md)).